### PR TITLE
Update the Readme and Changelog docs. Zowe Explorer 1.9.0 release

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <43951184+IgorCATech@users.noreply.github.com>

## Proposed changes

This PR updates both ReadMe and Changelog with 1.9.0 release notes. 

Milestone: 1.9.0

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `npm run vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [x] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
-## '1.8.0'
+## `1.9.0`
+
+- Added the Allocate Like feature. [#904](https://github.com/zowe/vscode-extension-for-zowe/issues/904). Thanks @katelynienaber
+- Added the ability to disable profile validation. [#922](https://github.com/zowe/vscode-extension-for-zowe/issues/922), [#953](https://github.com/zowe/vscode-extension-for-zowe/issues/953). Thanks @JillieBeanSim
```
Comment (zdmullen) [inline CHANGELOG.md:8]: I believe that 953 is supposed to be a separate bullet item. @zFernand0 ?

**Review by zdmullen — APPROVED** [review]
One comment, but I think it is okay if we release even the way it is. Thanks

**Comment by zFernand0** [comment]
Thank you for adding the readme information.
I'll split the bullet point and then we should be good to go! 😋

**Review by zFernand0 — APPROVED** [review]

**Review by zdmullen — APPROVED** [review]
LGTM! Thanks

**Pull request merged**